components/maskedtextbox/mask-prompt.md

```diff
@@ -97,27 +97,30 @@ By default, the `Value` of the component only includes the rules from the mask.
 }
 ````
 
-### Mask on Focus, Label and Placeholder
+### Mask on Focus, FloatingLabel and Placeholder
 
-The `MaskOnFocus` parameter lets you instruct the component to show the mask only when the user is about to type in the input - when it is focused. This lets you show the `Label` or `Placeholder` that you can set so you can provide an easier to read prompt first, before you show the actual format to your users.
+The `MaskOnFocus` parameter lets you instruct the component to show the mask only when the user is about to type in the input - when it is focused. This lets you show the [FloatingLabel]({%slug floatinglabel-overview%}) or `Placeholder` that you can set so you can provide an easier to read prompt first, before you show the actual format to your users.
 
-The `Label` will be rendered over the `Placeholder` and neither will be shown if there is a `Value` already. You should use the `Placeholder` over the `Label` if you do not want the animated effect or the increased height it causes.
-
-If `MaskOnFocus` is set to `false` (its default value), the `Label` will always be above the input and never inside (over) the input itself - the mask takes precedence - and the "floating" feature of the label will not be available.
+The [FloatingLabel]({%slug floatinglabel-overview%}) will take precedense over the `Placeholder` if the MaskedTextBox is not focused. Neither will be shown if there is a `Value` already. You should use the `Placeholder` over the [FloatingLabel]({%slug floatinglabel-overview%}) if you do not want the animated effect or the increased height it causes.
```

Comment:
```suggestion
The [FloatingLabel]({%slug floatinglabel-overview%}) will take precedence over the `Placeholder` if the MaskedTextBox is not focused. Neither will be shown if there is a `Value` already. You should use the `Placeholder` instead of a [FloatingLabel]({%slug floatinglabel-overview%}) if you do not want the animated effect or the increased height it causes.
```